src/main/webapp/app/exercises/shared/example-submission/example-submissions.component.html

```diff
@@ -8,13 +8,12 @@ <h4>
             </h4>
         </div>
         <div class="col-12 col-sm-auto d-flex">
-            <a
+            <button
```

Comment:
Those changes mean you no longer can use the button like a link (e.g. open in a new tab). Is that intended/necessary?

Reply:
I was, in fact, unaware you can also define a (click) on an a, I will revert that :+1:

Reply:
After changing the link still does not offer the usual link operations on right click, So either I still did something wrong or this is unfluenced by something else

Reply:
It doesn't work with click handlers but requires a router link.

Reply:
I see. Works like a charm now